internal/isoutil/isoutil.go

```diff
@@ -0,0 +1,238 @@
+package isoutil
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	diskfs "github.com/diskfs/go-diskfs"
+	"github.com/diskfs/go-diskfs/disk"
+	"github.com/diskfs/go-diskfs/filesystem"
+	"github.com/diskfs/go-diskfs/filesystem/iso9660"
+	"github.com/pkg/errors"
+)
+
+//go:generate mockgen -source=isoutil.go -package=isoutil -destination=mock_isoutil.go
```

Comment:
I think that it's more explicit. 
```suggestion
//go:generate mockgen -package=isoutil -destination=mock_isoutil.go . Handler
```